lib/Sema/TypeCheckDistributed.cpp

```diff
@@ -420,7 +420,10 @@ static bool checkDistributedTargetResultType(
   if (auto func = dyn_cast<FuncDecl>(valueDecl)) {
     resultType = func->mapTypeIntoContext(func->getResultInterfaceType());
   } else if (auto var = dyn_cast<VarDecl>(valueDecl)) {
-    resultType = var->getInterfaceType();
+    // Distributed computed properties are always getters,
+    // so get the get accessor for mapping the type into context:
+    auto getter = var->getAccessor(swift::AccessorKind::Get);
+    resultType = getter->mapTypeIntoContext(var->getInterfaceType());
```

Comment:
AccessorDecls inherits from `FuncDecl`, you can do something like before retrieving the result type:
```
if (auto *var = dyn_cast<VarDecl>(valueDecl)) {
  valueDecl = var->getAccessor(swift::AccessorKind::Get);
}
```

And the mapping would be handled by `FuncDecl` branch.

Reply:
Nice, thanks! Will do